.github/workflows/approved-for-ci-run.yml

```diff
@@ -66,13 +66,37 @@ jobs:
           GH_TOKEN: ${{ secrets.CI_ACCESS_TOKEN }}
         run: |
           HEAD="ci-run/pr-${PR_NUMBER}"
-          BODY="This Pull Request was create automatically to run CI pipeline for #${PR_NUMBER}.\n\nPlease do not alter or merge/close it.\n\nFeel free to comment the original PR."
+          cat << EOF > body.md
+            This Pull Request is created automatically to run the CI pipeline for #${PR_NUMBER}
 
-          ALREADY_CREATED=$(gh pr --repo "${GITHUB_REPOSITORY}" list --head "${HEAD}" --base "main" --json "number" --jq '.[].number')
+            Please do not alter or merge/close it.
+
+            Feel free to review/comment/discuss the original PR #${PR_NUMBER}.
```

Comment:
I am unable to compute what will happen to these indentations but I guess we'll test and see?

Reply:
It should be fine:

```
$ cat << EOF > body.md
  This Pull Request is created automatically to run the CI pipeline for #${PR_NUMBER}

  Please do not alter or merge/close it.

  Feel free to review/comment/discuss the original PR #${PR_NUMBER}.
EOF
$
$ cat body.md                                
  This Pull Request is created automatically to run the CI pipeline for #

  Please do not alter or merge/close it.

  Feel free to review/comment/discuss the original PR #.
```